front/src/modules/simulationResult/components/ManchetteWithSpaceTimeChart/ManchetteWithSpaceTimeChart.tsx

```diff
@@ -288,7 +287,7 @@ const ManchetteWithSpaceTimeChartWrapper = ({
     }
 
     // if not dragging, we check if we should start dragging
-    if (hoveredItem && 'pathId' in hoveredItem.element) {
+    if (hoveredItem && isSegmentPickingElement(hoveredItem.element)) {
```

Comment:
Probably we want to also allow dragging from a point? (`|| isPointPickingElement()`)

Reply:
added the point case !